.nvmrc

```diff
@@ -1 +1 @@
-20.10.0
```

Comment:
what's the reason for downgrading node version?

Reply:
Re-updated to `20.10.0` everywhere ✔️